python/ray/experimental/serve/examples/echo_fixed_packing.py

```diff
@@ -0,0 +1,45 @@
+"""
+Example showing fixed packing policy. The outputs from
+v1 and v2 will be coming according to packing_num specified!
+This is a packed round robin example
+"""
+import time
+
+import requests
+
+from ray.experimental import serve
+from ray.experimental.serve.utils import pformat_color_json
+
+
+def echo_v1(_):
+    return "v1"
+
+
+def echo_v2(_):
+    return "v2"
+
+
+# specify the router policy as FixedPacking with packing num as 5
```

Comment:
Does this mean the first five queries will go to echo:v1 and second batch of 5 queries go to echo:v2?

Reply:
Yes! exactly